javascript/javascript3/week1/Giphy/giphy.js

```diff
@@ -0,0 +1,30 @@
+const searchWord = document.querySelector('#searchWord');
+const searchButton = document.querySelector('#searchButton');
+const searchCount = document.querySelector('#searchCount');
+const imageDiv= document.querySelector('#imageDiv');
+const apiKey = 'S8WKPULrgpnM6GTyqIoRArXSpHrlgIgB';
+function getGiphyData() {
+    if(!searchWord.value) {
+        alert('Please provide a giphy name');
+    } else {
+        fetch(`http://api.giphy.com/v1/gifs/search?q=${searchWord}&api_key=${apiKey}&limit=${searchCount.value}`)
```

Comment:
the variable `searchWord` is an object with several fields, and not a string, so you can't pass it to the url. 
To get access to the input value, you simply specify the `value` field in `searchWord`, just like you did with `searchCount`: